Makefile

```diff
@@ -1,11 +1,24 @@
 # Copyright Contributors to the Open Cluster Management project
 
--include /opt/build-harness/Makefile.prow
+# VERSION defines the project version for the bundle.
+# Update this value when you upgrade the version of your project.
+# To re-generate a bundle for another specific version without changing the standard setup, you can:
+# - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
+# - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
+VERSION ?= $(shell cat COMPONENT_VERSION)-$(shell date +%s)
```

Comment:
Here are some thoughts about the VERSION. We may need to make some changes, but I don't want to hold this PR.
- For Github commit sha is preferred over date because we can correlate the the bundle to a specific commit.
- This version will also need to match the version in the CSV [here](https://github.com/stolostron/search-v2-operator-bundle/blob/main/bundle/manifests/search-v2-operator.clusterserviceversion.yaml#L317).  I'll need to check if the CSV accepts the postfix.

Reply:
Adding this change within a separate PR.